Docs -- callout quote v2 & Resource Card update (#1670)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/components/callout-quote.mdx

<AnchorLinks>

<AnchorLink>Overview</AnchorLink>
<AnchorLink>Modifiers</AnchorLink>
<AnchorLink>Content guidance</AnchorLink>
<AnchorLink>Resources</AnchorLink>
<AnchorLink>Feedback</AnchorLink>

</AnchorLinks>

## Overview

Callout quote enables a high impact moment in page content that highlights a key quote that reinforces the goals of the page. Callout quote is best used as a visual break (for example, between two [content sections](/components/content-section)) to vary the overall structure of the page and draw attention.

Callout quote can optionally include attribution information beneath the quote, as well as a link to relevant information.

![Default](../../images/pattern/callout-quote/callout-quote.jpg)

<br />

## Modifiers

### Color schemes

> Callout quote has additional color schemes to enable more flexibility in how callout content appears on the page. These include the default `regular` color scheme, as well as `inverse`, `layer`, `purple` , and `cyan`. When selecting a color scheme, keep in mind the adjacent colors from other components in the page layout.

Reviewer: @RichKummer this is super nit picky of me to ask, can we put inverse last in the list or after "layer"? 

```suggestion
Callout quote has additional color schemes to enable more flexibility in visual expression and variation. These include the default `regular` color scheme, as well as `layer`, `inverse`, `purple` , and `cyan`. When selecting a color scheme, keep in mind the adjacent colors from other components in the page layout.
```
